## lftp-config-dns-inet6_before_inet by Noèl Köthe <noel debian.org>
##
## DP: use inet6 dns before inet

--- lftp-3.4.7/lftp.conf.orig      2005-05-16 21:18:45.000000000 +0200
+++ lftp-3.4.7/lftp.conf   2005-05-16 21:19:06.000000000 +0200
@@ -87,3 +87,7 @@
 # set ftp:anon-pass "mozilla@"
 # set ftp:client ""
 # set http:user-agent "Mozilla/4.7 [en] (WinNT; I)"
+
+# try inet6 before inet
+set dns:order "inet6 inet"
+

